Packs/CommonScripts/Scripts/StringSimilarity/StringSimilarity.yml

```diff
@@ -21,7 +21,7 @@ outputs:
 - contextPath: StringSimilarity.SimilarityScore
   description: Similarity score - a value between 1.0 if the sequences are identical, and 0.0 if they have nothing in common.
 scripttarget: 0
-dockerimage: demisto/python3:3.10.12.68714
+dockerimage: demisto/python3:3.10.13.78960
 subtype: python3
```

Comment:
Make sure to adjust the comment as well (calculating the similarity between every string in 2 different arrays)